packages/edit-site/src/components/page-pages/index.js

```diff
@@ -257,11 +265,25 @@ export default function PagePages() {
 			{
 				header: __( 'Status' ),
 				id: 'status',
-				getValue: ( { item } ) =>
+				fieldType: ENUMERATION_TYPE,
```

Comment:
>I've added a fieldType prop, but probably should be merged with the current type prop used for filters

I didn't want to mess with the filters for now in case different types('date, filter, etc..`) broke something.

Reply:
If there's any, I'd rather bring them up sooner than later.

Reply:
Agree with this, I feel like type and fieldType are the same thing and we should just have one "type".